packages/store/package.json

```diff
@@ -34,7 +34,7 @@
     }
   },
   "scripts": {
-    "build": "pnpm run build:mud && pnpm run build:abi && pnpm run build:typechain && pnpm run build:js",
+    "build": "pnpm run generate-tightcoder && pnpm run build:mud && pnpm run build:abi && pnpm run build:typechain && pnpm run build:js",
```

Comment:
should this be done differently? (the only other precedent of custom codegen is in `cli`, but that is test-only)